skbio/diversity/alpha/_pd.py

```diff
@@ -50,7 +50,8 @@ def _faith_pd(counts_by_node, branch_lengths):
     return (branch_lengths * (counts_by_node > 0)).sum()
 
 
-def faith_pd(counts, taxa=None, tree=None, validate=True, otu_ids=None):
+@params_aliased([("taxa", "otu_ids", "0.6.0", True)])
+def faith_pd(counts, taxa=None, tree=None, validate=True):
```

Comment:
We now need to make `taxa` and `tree` mandatory parameters. Same for other functions.